src/components/DateRangePicker.jsx

```diff
@@ -345,6 +345,11 @@ export default class DateRangePicker extends React.Component {
             withFullScreenPortal={withFullScreenPortal}
             onDatesChange={onDatesChange}
             onFocusChange={onFocusChange}
+            focusDayPicker={() => {
```

Comment:
I believe this behavior should only happen when withPortal or withFullscreenPortal is used. This is not the behavior I want for other situations. I also feel like this code belongs a layer down in the `DayPicker` itself, not here.